fingertip/plugins/reboot.py

```diff
@@ -0,0 +1,18 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2024 Red Hat, Inc., see CONTRIBUTORS.
+
+def main(m):
+    if 'wait_for_running' not in m.hooks:
+        m.log.critical(
```

Comment:
Please downgrade that to a `.warning()` and drop the RuntimeError. I don't see how not having it is anything more than a missed performance opportunity.

Reply:
Done.